test/runtests.jl

```diff
@@ -129,6 +130,10 @@ include("subdir/file3.jl")
 include(joinpath(@__DIR__, "subdir", "file4.jl"))
 otherfile = "file5.jl"
 include(otherfile)
+macro mac$(fbase)()
```

Comment:
The first code update changes the `using_mac()` definition, and the `$(fbase)1()` definition, which is called in the macroexpansion. If the former is updated before the latter (which had a 50% chance before this PR), then the macroexpansion will return 1, and the test will fail.

It's not a great test, admittedly, since it might have passed before this PR anyway. Do you have a better suggestion?